src/app/secure-data-environment/sde-sign-in/sde-sign-in.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
```

Comment:
Possibly combine with equivalent component in sde-admin-ui?

Reply:
I've now changed this to use a shared component from `sde-resources`